modules/brushingandlinking/include/modules/brushingandlinking/brushingandlinkingmanager.h

```diff
@@ -55,6 +55,7 @@ class IVW_MODULE_BRUSHINGANDLINKING_API BrushingAndLinkingManager {
     void remove(const BrushingAndLinkingInport* src);
 
     bool isFiltered(size_t idx) const;
+    bool isFilteredByOther(size_t idx, const inviwo::BrushingAndLinkingInport* src) const;
```

Comment:
I think that the BrushingAndLinkingManager should be responsible for pausing/resuming the filtering instead of the brushing and linking ports. 
Then we would not need this extra function?

Reply:
how does the manager know which event producer that is paused? this seems like a local state, why should we manage it globally?